schemas/test/en/test_currency.json

```diff
@@ -93,10 +93,93 @@
+                                        "currency": "GBP",
+                                        "decimal_places": 2,
+                                        "maximum": {
+                                            "value": 10000
+                                        },
+                                        "minimum": {
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-usd",
+                                        "label": "How much did you spend?",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "USD",
+                                        "decimal_places": 2,
+                                        "maximum": {
+                                            "value": 10000
+                                        },
+                                        "minimum": {
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-eur",
+                                        "label": "How much did you spend?",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "EUR",
+                                        "decimal_places": 2,
+                                        "maximum": {
+                                            "value": 10000
+                                        },
+                                        "minimum": {
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-jpy",
+                                        "label": "How much did you spend?",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "JPY",
+                                        "maximum": {
+                                            "value": 1000000
+                                        },
+                                        "minimum": {
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-min-max-range",
+                                        "label": "How much did you spend? (Max range)",
```

Comment:
Max range -> Min range?

Reply:
Changed to Min Range. Should I put min max range since it is both.